edalize/templates/libero/libero-run.tcl.j2

```diff
@@ -0,0 +1,14 @@
+# This file is generated by Edalize.
+# Microsemi Tcl Script
+# Libero
+
+run_tool -name {SYNTHESIZE}
```

Comment:
This needs to start with `open_project {path/to/project/generated/by/first/tcl/file}`

Reply:
Done.